--- coova-chilli-1.2.4/src/chilli_limits.h~	2010-08-24 13:29:08.000000000 +0200
+++ coova-chilli-1.2.4/src/chilli_limits.h	2010-09-30 12:21:20.000000000 +0200
@@ -75,7 +75,7 @@
 #define MACSTRLEN                         17
 #define MS2SUCCSIZE                       40 /* MS-CHAPv2 authenticator response as ASCII */
 #define DATA_LEN                        1500 /* Max we allow */
-#define USERNAMESIZE                     256 /* Max length of username */
+#define USERNAMESIZE                     255 /* Max length of username */
 #define CHALLENGESIZE                     24 /* From chap.h MAX_CHALLENGE_LENGTH */
 #define USERURLSIZE                      256 /* Max length of URL requested by user */
 
