--- coova-chilli-1.2.4/src/Makefile.am~	2010-08-24 13:29:08.000000000 +0200
+++ coova-chilli-1.2.4/src/Makefile.am	2010-10-14 13:00:28.640702861 +0200
@@ -24,6 +24,7 @@
 iphash.c lookup3.c sfhash.c lookup.c system.h cmdsock.c util.c \
 garden.c ssl.c dns.c session.c pkt.c chksum.c net.c \
 ms_chap.c options.c statusfile.c conn.c sig.c
+libchilli_la_LIBADD = $(top_builddir)/bstring/libbstring.la ${LIBRT}
 
 AM_CFLAGS = -D_GNU_SOURCE -Wall -Werror -fno-builtin -fno-strict-aliasing \
   -O2 -fomit-frame-pointer -funroll-loops -pipe \
